### PR TITLE
adding content for home page, pricing, get started, contact us

### PR DESCRIPTION
**Pull request by jamesthebrooks**
adding content for home page, pricing, get started, contact us

### CODE REVIEW

File: .eleventy.js [hunk 1]
```diff
@@ -13,6 +13,68 @@ const pluginMermaid = require("@kevingimbel/eleventy-plugin-mermaid");
...
+    const filePath = path.join(directoryPath, file);
+    const fileStat = fs.statSync(filePath);
+    if (fileStat.isFile() && fileTypes.includes(path.extname(file).toLowerCase())) {
+      matchingFiles.push(file);
+    }
+  }
+  return matchingFiles;
+}
+
+function getFrontMatterSync(filePath) {
```
Comment (sknep) [inline .eleventy.js:46]: please annotate these special functions with what they're intended to be used for, for future generations :)

File: .eleventy.js [hunk 2]
```diff
@@ -191,6 +259,18 @@ module.exports = function (config) {
   config.addFilter("markdown", (content) => {
     return markdownLibrary.render(content);
   });
+  markdownLibrary.renderer.rules.link_open = function (tokens, idx, options, env, self) {
```
Comment (sknep) [inline .eleventy.js:262]: i thought you added this in the last PR. is this similar/different?

File: _data/support-packages.yaml [hunk 3]
```diff
@@ -6,7 +6,7 @@
   image: basic-support.svg
 - package_name: Basic <br>Support
   description: >
-    This is the default support level for new cloud.gov agreements. This package includes all services provided in Platform Support, plus 10hrs of support time that our engineers can devote to resolving issues with your specific applications.
+    This is the default support level for new cloud.gov agreements. This package includes all services provided in Platform Support, plus 10hrs of support time that our engineers can devote to resolving issues with your specific applications.<br /><br />
```
Comment (sknep) [inline _data/support-packages.yaml:9]: why line breaks here? I don't want to insert HTML into YAML fields. If we need vertical space, CSS should do that for us.

File: content/resources/Contingency-plan-exercise-template.docx
Comment (sknep) [inline content/resources/Contingency-plan-exercise-template.docx:1]: are these documents linked to from these files? why are they included?

File: .eleventy.js (hunk 1 above)
Comment (sknep) [reply .eleventy.js:49]: thanks!

File: _data/support-packages.yaml (hunk 3 above)
Comment (jamesthebrooks) [reply _data/support-packages.yaml:9]: It's a carry over from the cg-site here:  https://github.com/cloud-gov/cg-site/blob/main/_data/support-packages.yml

I'll update to a CSS solution.

File: content/resources/Contingency-plan-exercise-template.docx
Comment (jamesthebrooks) [reply content/resources/Contingency-plan-exercise-template.docx:1]: These weren't in the assets folder to begin with in cg so they didn't make it in.  I just updated the code base to move everything inside /resources to /assets/documents.  I updated paths where I saw the files referenced and created redirects for anyone who might be trying to link to them directly.  Likely some of these files can be permanently deleted as well, but this was not addressed in the PDFs pull request.

File: content/resources/cloudgov-vendors-2019.pdf
[the remark this answers is not in the document]
Comment (jamesthebrooks) [reply content/resources/cloudgov-vendors-2019.pdf:1]: Similar comment as above.

**Comment by sknep** [comment]
is this ready for review?

File: content/resources/Contingency-plan-exercise-template.docx
Comment (sknep) [reply content/resources/Contingency-plan-exercise-template.docx:1]: OK lets either drop all of them from this PR or go through them and pick which are actually needed. I don't want to bring over more files we don't need. Can you tell if these are directly linked to in the markdown content you're pulling over?

File: _assets/documents/casestudy-fec.pdf
Comment (sknep) [inline _assets/documents/casestudy-fec.pdf:1]: redirect to https://cloud.gov/docs/customer-stories/fec/

File: content/redirects/cloudgov-customers.html [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+permalink: /resources/cloud-gov-customers.pdf/index.html
+redirect: /assets/documents/cloud-gov-customers.pdf
```
Comment (sknep) [inline content/redirects/cloudgov-customers.html:3]:
```suggestion
redirect: /docs/customer-stories/epa-airnow-gov/
```

File: content/redirects/cloudgov-overview-2018.html [hunk 5]
```diff
@@ -0,0 +1,5 @@
+---
+permalink: /resources/cloudgov-overview-2018.pdf/index.html
+redirect: /assets/documents/cloudgov-overview-2018.pdf
```
Comment (sknep) [inline content/redirects/cloudgov-overview-2018.html:3]:
```suggestion
redirect: /docs/overview/cloudgov-benefits/
```

File: content/redirects/Contingency-plan-exercise-template.pdf.html [hunk 6]
```diff
@@ -0,0 +1,5 @@
+---
+permalink: /resources/Contingency-plan-exercise-template.pdf/index.html
+redirect: /assets/documents/Contingency-plan-exercise-template.pdf
```
Comment (sknep) [inline content/redirects/Contingency-plan-exercise-template.pdf.html:3]:
```suggestion
redirect: /docs/ops/moved/
```

File: content/redirects/cloudgov_cost_estimator.html [hunk 7]
```diff
@@ -0,0 +1,5 @@
+---
+permalink: /resources/cloudgov_cost_estimator.ods/index.html
+redirect: /assets/documents/cloudgov_cost_estimator.ods
```
Comment (sknep) [inline content/redirects/cloudgov_cost_estimator.html:3]:
```suggestion
redirect: /pricing
```

File: content/sign-up/index.md [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+</div>
+<div class="tablet:grid-col-1"></div>
+<div class="tablet:grid-col usa-prose usa-section--sidebar-links">
+<h4>Join the communities</h4>
+        <ul>
+            <li>
+            <a href="https://docs.google.com/forms/d/1vcsvQ64qt5mYNyVajcwtYDRMqEOyPzsXZBGM5c4_BD8/edit">The public DevOps channel on TTS's Slack (sign up at using this form).</a>TTS Slack
+            </li>
+            <li>
+            <a href="https://www.cloudfoundry.org/community/">Cloud Foundry communities (Slack, newsletters, mail lists)</a>
```
Comment (sknep) [inline content/sign-up/index.md:102]: does this open in a new window in the built version?

**Review by sknep — APPROVED** [review]
thanks!

**Pull request merged**